job/job.go

```diff
@@ -176,11 +165,19 @@ func (j Job) MarkDeviceActive(ctx context.Context) {
 	}
 }
 
-func (j *Job) setup(d packet.Discovery) error {
+func (j *Job) setup(ctx context.Context, d packet.Discovery) (context.Context, error) {
```

Comment:
definitely need to document the returned ctx here.

Reply:
added

Reply:
thanks